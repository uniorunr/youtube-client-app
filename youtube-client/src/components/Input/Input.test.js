import inputComponent from './Input';

describe('inputComponent', () => {
  const main = document.createElement('main');
  document.body.appendChild(main);

  it('should be instance of Function', () => {
    expect(inputComponent).toBeInstanceOf(Function);
  });

  it('should be rendered correctly', () => {
    inputComponent();
    expect(document.body.innerHTML).toMatchSnapshot();
  });
});
