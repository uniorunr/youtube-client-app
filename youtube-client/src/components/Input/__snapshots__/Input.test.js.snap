// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`inputComponent should be rendered correctly 1`] = `"<main><section class=\\"input-wrapper\\"><input class=\\"search-input\\" placeholder=\\"Search...\\" type=\\"search\\"></section></main>"`;
